README.md

```diff
@@ -1,37 +1,44 @@
 # 📖 OSPOlogy: The Study of OSPOs
 
-The [TODO Group](https://todogroup.org) hosts a monthly meeting and newsletter to study and discuss the status of OSPOs.
+✨ **`OSPOlogy` hosts the study and open community communication of industry open source program offices.**
 
-OSPOlogy is open to all, from folks just starting their OSPO journey to students to even seasoned OPSOers!
+## 🚀 Our Mission
 
-## Community Meeting (monthly)
+The [TODO Group](https://todogroup.org) is an open community of practitioners who aim to create and share knowledge, collaborate on practices, tools, and other ways to run successful and effective Open Source Program Offices or similar Open Source initiatives. TODO Group is formed by its [Community participants](https://todogroup.org/community) and [General Members](https://todogroup.org/members).
 
-Everyone is welcome to attend the monthly TODO Group OSPOlogy meeting: https://community.linuxfoundation.org/todo-group/
+## 🦺 Community
 
-## TODO EU Chapter Sync Meetings (monthly)
+To paritipate in OSPO related discussions please [submit questions](https://github.com/todogruop/ospology/discussions) or join the [TODO Group Slack](https://slack.todogroup.org).
 
-The TODO EU chapter was formed by TODO Members based in Europe. This chapter was created with the aim of working together to improve OSPO adoption and education within Europe and discuss with the broader community the challenges European organizations face when implementing an open source program.
+### 🌎 North America Community Meeting
 
```

Comment:
TODO Community meetings structure changes a bit from what is has been described here, let me explain this better 🙂 They are structured into:

* [OSPOlogy community meetings](https://github.com/todogroup/ospology/tree/main/meetings#ospology-monthly-meetings): These are "General and public monthly meetings of the TODO Group and wider OSPO community, everyone is welcome to attend and participate". These meetings have always been a featured topic with special guests and the last 15 mins are reserved for community discussions. all these meetings are recorded, edited, and recorder on Youtube OSPOlogy Channel, so everyone across the globe can have access to it 👍 https://github.com/todogroup/ospology/tree/main/meetings#event-page-and-past-sessions

* TODO Chapters: These are location-based meetings and they do not follow the same structure as OSPOlogy community meetings. The community put topics of matter into an agenda and we have discussions the entire hour. We currently have [Europe Sync meetings](https://github.com/todogroup/ospology#todo-eu-chapter-sync-meetings-monthly) and [APAC Sync meetings](https://github.com/todogroup/ospology/discussions/67). Both are available in the community page calendar: https://todogroup.org/community/